### ISSUE TITLE
pretrained model

### ISSUE THREAD

**Issue by Ai-is-light** [fandulu/DD-Net#2]
would you mind sharing your pretrained model or demo for more exploring

**Comment by fandulu** [comment]
Hi, I do not offer the pretrained model since both of the data and model are very small, it only takes a few minutes to train them. And, I suppose the seed setting may affect the results sometimes, but more or less you can obtain the performance as I claimed. The demo link is given in https://colab.research.google.com/drive/19gq3bUigdxIfyMCoWW93YhLEi1KQlBit, just download the data from my /data folder and upload it for running.

**Comment by Ai-is-light** [comment]
thx, I try, good work

**Comment by Ai-is-light** [comment]
DD_Net.load_weights('weights/fine_heavy.h5')
where can I find or get the fine_heavy.h5

**Comment by fandulu** [comment]
I got your point. Sorry for this confusion setting. I use it for testing purpose. It can be ignored and train the model from scratch.

**Comment by Ai-is-light** [comment]
@fandulu 
I got the keypoint from the data folder from your support and they are, as following:
[[ 0.286 -0.377]
 [ 0.286 -0.066]
 [ 0.261 -0.451]
 [ 0.189 -0.332]
 [ 0.392 -0.362]
 [ 0.256  0.002]
 [ 0.295  0.032]
 [ 0.106 -0.269]
 [ 0.512 -0.387]
 [ 0.257  0.208]
 [ 0.244  0.262]
 [-0.05  -0.218]
 [ 0.638 -0.42 ]
 [ 0.419  0.189]
 [ 0.111  0.476]]
I want to kwon how to deal with the keypoint to get the format like above. If I have the new keypoint from the openpose? It is each keypoint's x and y coordinate subtract the keypoint center(x_kp_new = x_kp_old - all_x_kp_center)/all_x_kp_center?
I try to get the information from the JHMDB'paper, but I did not find clearly information about it

**Comment by Ai-is-light** [comment]
pose_path = './joint_positions.mat'
pose = read_pose(pose_path)
print(len(pose))
print(pose[0])

[[ 0.286 -0.377]
[ 0.286 -0.066]
[ 0.261 -0.451]
[ 0.189 -0.332]
[ 0.392 -0.362]
[ 0.256 0.002]
[ 0.295 0.032]
[ 0.106 -0.269]
[ 0.512 -0.387]
[ 0.257 0.208]
[ 0.244 0.262]
[-0.05 -0.218]
[ 0.638 -0.42 ]
[ 0.419 0.189]
[ 0.111 0.476]]

**Comment by fandulu** [comment]
In my project, it is the center of the image other than the center of poses since the global coordinates are used for calculating the motion features. However, it is really depended on your target task, like the paper explained, when your target does not correspond to global trajectories, use pose center is also fine.

**Comment by Ai-is-light** [comment]
@fandulu yeah, my target dose try to deal with the global trajectories, I just would like to make sure it. I would be better if you support one demo or example about the new video as input, which have been processed to the pose format.
Thanks

**Comment by Ai-is-light** [comment]
@fandulu and by the way, the print(pose[0])

[[ 0.286 -0.377]
[ 0.286 -0.066]
[ 0.261 -0.451]
[ 0.189 -0.332]
[ 0.392 -0.362]
[ 0.256 0.002]
[ 0.295 0.032]
[ 0.106 -0.269]
[ 0.512 -0.387]
[ 0.257 0.208]
[ 0.244 0.262]
[-0.05 -0.218]
[ 0.638 -0.42 ]
[ 0.419 0.189]
[ 0.111 0.476]]
, it is computed by (x-center_x)/width, the (x,y) is the pose's position and the (center_x, center_y)is the image's center and the (width, height)is the image's width and height?
